.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,70 @@
+name: CI Build
+
+on:
+  - push
```

Comment:
Do you want to run it also when you merge into main?

Reply:
Yes, but I think it is run with this setting. Merge to main also creates a new commit push, if I am not wrong. Or did you have something else in mind?